app/src/main/java/org/wikipedia/descriptions/DescriptionEditActivity.java

```diff
@@ -32,21 +34,22 @@
     private static final String EXTRA_HIGHLIGHT_TEXT = "highlightText";
     private static final String EXTRA_TRANSLATION_SOURCE_DESCRIPTION = "translationSourceDescription";
     private static final String EXTRA_TRANSLATION_SOURCE_LANGUAGE_CODE = "translationSourceLanguageCode";
+    private static final String EXTRA_PAGE_EXTRACT = "extract";
     private static final String EXTRA_INVOKE_SOURCE = "invokeSource";
     private InvokeSource invokeSource;
     private ExclusiveBottomSheetPresenter bottomSheetPresenter = new ExclusiveBottomSheetPresenter();
 
     public static Intent newIntent(@NonNull Context context,
                                    @NonNull PageTitle title,
+                                   @NonNull String extract,
                                    @Nullable String highlightText,
-                                   boolean reviewEnabled,
                                    @Nullable CharSequence translationSourceDescription,
                                    @Nullable String translationSourceLanguageCode,
                                    @NonNull InvokeSource invokeSource) {
```

Comment:
Had to reduce a parameter to introduce a new one, since we can derive reviewEnabled, chose that.